scripts/generate_skus.go

```diff
@@ -0,0 +1,128 @@
+		"regional",
+		"nearline",
+		"coldline",
+		"archive",
+	}
+
+	for region, location := range regions {
+		fmt.Printf("{\n")
+		fmt.Printf("	Region: repository.Region(%q),\n", region)
+		fmt.Printf("	Location: Location{\n")
+		fmt.Printf("		Latitude:  %f,\n", location.Latitude)
+		fmt.Printf("		Longitude: %f,\n", location.Longitude)
+		fmt.Printf("	},\n")
+		fmt.Printf("	StorageClasses: []RegionConfigurationStorageClass{\n")
+		for _, class := range classes {
+			for _, dualRegion := range []bool{false, true} {
+				storageSku := ""
+				earlyDeleteSku := ""
+				ttl := 0
+				switch class {
+				case "nearline":
+					ttl = 1
+				case "coldline":
+					ttl = 3
+				case "archive":
+					ttl = 12
+				}
+				for _, sku := range res.Skus {
+					if sku.Category.ResourceFamily == "Storage" &&
+						sku.Category.UsageType == "OnDemand" &&
+						len(sku.ServiceRegions) == 1 && sku.ServiceRegions[0] == region &&
+						(strings.ToLower(sku.Category.ResourceGroup) == class+"storage" || sku.Category.ResourceGroup == "MultiRegionalStorage" && dualRegion && class == "regional") &&
+						(strings.Contains(sku.Description, "Dual-region") && dualRegion || !strings.Contains(sku.Description, "Dual-region") && !dualRegion) {
+						if strings.Contains(sku.Description, "(Early Delete)") {
+							earlyDeleteSku = sku.SkuId
+						} else {
+							storageSku = sku.SkuId
+						}
+					}
+				}
+				other := ""
+				if earlyDeleteSku != "" {
+					other = fmt.Sprintf(", EarlyDeleteSKU: %q, MinTTL: %d", earlyDeleteSku, ttl)
+				}
+				if storageSku != "" {
+					fmt.Printf("		{StorageClass: %q, DualRegion: %v, StorageSKU: %q%s},\n", strings.ToUpper(class), dualRegion, storageSku, other)
+				}
+			}
+		}
+		fmt.Printf("	},\n")
+		fmt.Printf("},\n")
+	}
+
+	fmt.Printf("{\n")
+	fmt.Printf("	Region: repository.Region(%q),\n", "eu")
+	fmt.Printf("	MultiRegion: true,\n")
+	fmt.Printf("	StorageClasses: []RegionConfigurationStorageClass{\n")
+	for _, class := range classes {
+		storageSku := ""
+		earlyDeleteSku := ""
+		ttl := 0
+		switch class {
+		case "nearline":
+			ttl = 1
+		case "coldline":
+			ttl = 3
+		case "archive":
+			ttl = 12
+		}
+		for _, sku := range res.Skus {
+			if sku.Category.ResourceFamily == "Storage" &&
+				sku.Category.UsageType == "OnDemand" &&
+				len(sku.ServiceRegions) == 1 && sku.ServiceRegions[0] == "europe" &&
+				(strings.ToLower(sku.Category.ResourceGroup) == class+"storage" || sku.Category.ResourceGroup == "MultiRegionalStorage" && class == "regional") {
+				if strings.Contains(sku.Description, "(Early Delete)") {
+					earlyDeleteSku = sku.SkuId
+				} else {
+					storageSku = sku.SkuId
+				}
+			}
+		}
+		other := ""
+		if earlyDeleteSku != "" {
+			other = fmt.Sprintf(", EarlyDeleteSKU: %q, MinTTL: %d", earlyDeleteSku, ttl)
+		}
+		fmt.Printf("		{StorageClass: %q, DualRegion: %v, StorageSKU: %q%s},\n", strings.ToUpper(class), false, storageSku, other)
+	}
+	fmt.Printf("	},\n")
+	fmt.Printf("},\n")
+
+}
```

Comment:
**question**: is there possibility to assert data generation; for example _count entries has minimal value 20_?

Reply:
The human can check on completeness. When he does the review on PR